source/Mission.cpp

```diff
@@ -208,6 +208,10 @@ void Mission::Load(const DataNode &node)
 			location = ASSISTING;
 		else if(child.Token(0) == "boarding")
 			location = BOARDING;
+		else if(child.Token(0) == "shipyard")
+			location = SHIPYARD;
+		else if(child.Token(0) == "outfitter" && child.Size() == 1)
```

Comment:
What if the size of the token isn't 1? This check seems to be unique to this line?

Reply:
Want to avoid conflict with this other change I proposed where `outfitter` can be used to describe and objective. So if there is only one child it is setting the location, otherwise it is setting an objective.

Reply:
Other PR
https://github.com/endless-sky/endless-sky/pull/7761

Reply:
Idk if the same keyword should do both things honestly.

Reply:
I think the "objectives" in your other PR perhaps shouldn't go at this level anyway, or otherwise should have a parent node identifying them as objectives, so let's not make this entry weird in this PR to support that.

Reply:
@warp-core Wouldn't that break the pattern with existing objectives like npc, waypoint, destination & cargo?

Reply:
Changing other PR closing this conversation out